SPECS/dmidecode/dmidecode.spec

```diff
@@ -1,6 +1,6 @@
 Summary:	Tool to analyze BIOS DMI data
 Name:		dmidecode
-Version:	3.3
```

Comment:
Any concerns with compatibility differences between 3.3 and 3.5?  Things to check are the changelog for dmidecode and anything that may be dependent on dmidecode.

This is one where we might have preferred the patch to the upgrade.  Also I'm a little surprised that the autoupgrade pipeline even upgraded this one...

Reply:
dmidecode is not in any stable version of Ubuntu, and would assume it is not in most stable linux distribution for now. While I don't think it will break anything, I agree we might want to patch Rakshaa Viswanathan. Sadly, release note is not that verbose : https://savannah.nongnu.org/news/?id=10322